pkgs/development/python-modules/langchain-tests/default.nix

```diff
@@ -19,6 +19,7 @@
   pytest,
 
   # tests
+  blockbuster,
```

Comment:
```suggestion
```

Reply:
blockbuster is not used in the langchain-tests expression, only in the langchain-anthropic expression

Reply:
You're right -- it builds fine if I remove `blockbuster`. I wonder what it's doing in `uv.lock`.